.pre-commit-config.yaml

```diff
@@ -98,3 +98,12 @@ repos:
       - id: isort
         name: isort (pyi)
         types: [pyi]
+  - repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+      - id: black
+        # It is recommended to specify the latest version of Python
+        # supported by your project here, or alternatively use
+        # pre-commit's default_language_version, see
+        # https://pre-commit.com/#top_level-default_language_version
+        language_version: python3.9
```

Comment:
Hmm, is there any way to make this more flexible? I'm developing with 3.11...

Reply:
I can try this with the default language version, I don’t think it actually changes the formatting. I just copy/pasted this snippet from the black docs.